blueprints/test-framework-detector.js

```diff
@@ -12,7 +14,12 @@ module.exports = function(blueprint) {
     if ('ember-cli-qunit' in dependencies) {
       type = 'qunit';
     } else if ('ember-cli-mocha' in dependencies) {
-      type = 'mocha';
+      var checker = new VersionChecker({ project: this.project });
+      if (fs.existsSync(this.path + '/mocha-0.12-files') && checker.for('ember-cli-mocha', 'npm').satisfies('>=0.12.0')) {
```

Comment:
Why is the check for `/mocha-0.12-files` necessary`? Shouldn't this be the case when the addon of this version is installed?